packages/lpa-questionnaire-web-app/src/controllers/upload-plans.js

```diff
@@ -17,6 +17,7 @@ exports.getUploadPlans = (req, res) => {
 
   res.render(VIEW.UPLOAD_PLANS, {
     appeal: getAppealSideBarDetails(req.session.appeal),
+    appealReply: req.session.appealReply,
```

Comment:
Passing the whole appealReply for this seems heavy. Perhaps just pass the ID and rename the variable accordingly?